src/arr/trove/cmdline.arr

```diff
@@ -2,11 +2,11 @@
 
 provide {
   file-name: file-name,
-  args: other-args,
+  args: args,
   Number: read-number,
-  Bool: read-bool,
-  String: read-string,
-  Custom: read-custom,
+  Bool: Bool,
+  String: String,
+  Custom: Custom,
```

Comment:
This seems weird – is this _required_ by this commit or was it done for other reasons? It seems like this commit should be making this kind of renaming less necessary.